atomix/cluster/src/main/java/io/atomix/raft/roles/InactiveRole.java

```diff
@@ -78,66 +77,72 @@ public CompletableFuture<ConfigureResponse> onConfigure(final ConfigureRequest r
   @Override
   public CompletableFuture<InstallResponse> onInstall(final InstallRequest request) {
     logRequest(request);
-    return Futures.completedFuture(
+    final io.atomix.raft.protocol.InstallResponse result =
```

Comment:
:wrench: I think these types declarations can be cleaned up by either removing the package qualifier or just using `var`s

Reply:
My bad, I just used the automated inlining from IntelliJ